modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -260,14 +272,16 @@ export default class LargeVideoManager {
                 // TODO: when this class is converted to a function react component,
                 // use a custom hook to update a local track streaming status.
                 if (this.videoTrack?.jitsiTrack?.getSourceName() !== videoTrack?.jitsiTrack?.getSourceName()) {
```

Comment:
Previously this.videoTrack is not updated when videoTrack is undefined, which makes the following code to execute over and over again.